src/main/java/com/nicoislost/ZoomO.java

```diff
@@ -1,23 +1,40 @@
 package com.nicoislost;
 
-import com.nicoislost.inputs.KeyBinds;
+import com.nicoislost.inputs.KeyBind;
 import com.nicoislost.owo.ZoomOConfig;
 import com.nicoislost.util.ModRegistries;
 
 import net.fabricmc.api.ClientModInitializer;
 import net.minecraft.client.MinecraftClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
You don't need the logger, but it made it easier to verify the workflow was working correctly